src/fragments/lib/predictions/js/getting-started.mdx

```diff
@@ -32,14 +47,9 @@ import rn1 from '/src/fragments/lib/common/react-native/import_configuration.mdx
 
 ```javascript
 import { Amplify } from 'aws-amplify';
-import {
-  Predictions,
-  AmazonAIPredictionsProvider
-} from '@aws-amplify/predictions';
 import awsconfig from './aws-exports';
```

Comment:
For v6 docs, I think we're using `import amplifyconfig from './amplifyconfiguration.json';`

Reply:
Wild! Didn't realize this directory was changing for v6. Will update.